installutils/helminstall/installer.go

```diff
@@ -37,16 +33,14 @@ type installer struct {
 	out          io.Writer
 }
 
-func MustInstaller() types.Installer {
-	cfg, err := kubeutils.GetConfig("", "")
-	if err != nil {
-		log.Fatal(err)
+type InstallerFactory func(helmClient types.HelmClient) types.Installer
```

Comment:
could we do away with this factory as well per the discussion in a previous comment, and simply expose `NewInstaller`?

Reply:
ah missed this one